app/controllers/application_controller.rb

```diff
@@ -83,7 +83,10 @@ def t_and_c
     decorate_user_if_present
 
     terms_and_conditions = TermsAndConditions.last
-    terms_and_conditions = terms_and_conditions[:description]
+    terms_and_conditions = page_content = {
+      :en => terms_and_conditions[:description],
+      :es => terms_and_conditions[:spanish_description]
```

Comment:
The page crashes if the db does not have a terms and conditions in spanish, would recommend adding an edge case handler here. Just render the toc in english if we don't have a spanish version.

```suggestion
      :es => terms_and_conditions[:spanish_description] || terms_and_conditions[:description]
```